src/daemon.rs

```diff
@@ -117,6 +117,26 @@ struct NetworkInfo {
     relayfee: f64, // in BTC/kB
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+struct MempoolFees {
+    base: f64,
+    #[serde(rename = "effective-feerate")]
+    effective_feerate: f64,
+    #[serde(rename = "effective-includes")]
+    effective_includes: Vec<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct MempoolAcceptResult {
+    txid: String,
+    wtxid: String,
+    allowed: Option<bool>,
```

Comment:
It might be a good idea to document what `None` means for allowed.

This will be serialized as no allowed key when sent to the client... which is weird.

Reply:
Looks like it will be `None` when the transaction was not even validated. (ie. because a parent in the same request already returned false for allowed....

We could collapse all None into false imo...... but I mean, however you want to do it is fine.

Reply:
Apparently you can verify unconfirmed packages but you need to place them in the array in dependency order, so the parent always comes before the child.

It's also kind of weird that allowed might be false because of having too high of a fee.

Depending on what we want to do here, it might be a good idea to add the maxfeerate option as well. (default 0.1 BTC/kvB)

Reply:
> This will be serialized as no allowed key when sent to the client... which is weird.

you would think so, but it actually seems to get serialized as `{ ..., allowed: null, ... }`.

Reply:
Collapsing to false does seem sensible, but if the key being omitted conveys some information that isn't apparent otherwise (i.e. that the transaction is disallowed because an ancestor failed validation), then I'm reluctant to remove that.

Reply:
> Depending on what we want to do here, it might be a good idea to add the maxfeerate option as well. (default 0.1 BTC/kvB)

yes, absolutely